sdk/messaging/azservicebus/internal/amqpLinks.go

```diff
@@ -118,22 +118,42 @@ func (links *amqpLinks) ManagementPath() string {
 
 // recoverLink will recycle all associated links (mgmt, receiver, sender and session)
 // and recreate them using the link.linkCreator function.
-func (links *amqpLinks) recoverLink(ctx context.Context) error {
+func (links *amqpLinks) recoverLink(ctx context.Context, theirLinkRevision *uint64) error {
```

Comment:
This is a pointer to disambiguate zero from no revision?

Reply:
Yeah, just to make it more obvious.

I actually do start the link revision from 1 to avoid this but I felt like it was just too subtle in the source code.